pkgs/tools/misc/goss/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "goss";
+  version = "0.3.16";
+
+  src = fetchFromGitHub {
+    owner = "aelsabbahy";
+    repo = "goss";
```

```suggestion
    repo = pname;
```

Can use `pname` here or just leave it as-is

---

Nice !